supermarq/features.py

```diff
@@ -0,0 +1,163 @@
+from typing import Union
+
+import cirq
+import networkx as nx
+import numpy as np
+import qiskit
+import supermarq as sm
+
+
+def compute_communication(circuit: Union[cirq.Circuit, qiskit.circuit.QuantumCircuit]) -> float:
+    """
+    communication feature = circuit's average qubit degree / degree of a complete graph
```

Comment:
Change docstring to be in the style of Google's recommendation: https://google.github.io/styleguide/pyguide.html